lib/ex_unit/lib/ex_unit.ex

```diff
@@ -360,6 +360,23 @@ defmodule ExUnit do
     ExUnit.Runner.run(options, nil)
   end
 
+  @doc """
+  Rerun the given test modules synchronously.
+
+  Returns a map containing the total number of tests, the number
+  of failures, the number of excluded tests and the number of skipped tests.
+  """
+  @spec rerun(list(atom())) :: suite_result()
+  def rerun(modules) do
+    for module <- modules do
+      ExUnit.Server.add_sync_module(module)
+    end
+
+    _ = ExUnit.Server.modules_loaded()
+    options = persist_defaults(configuration())
+    ExUnit.Runner.run(options, nil)
```

Comment:
I think we can call `run` instead?

Reply:
But now I am thinking if we should instead simply make the API be `ExUnit.run(additional_modules \\ [])`? Instead of adding `rerun`?

Reply:
Yeap I think that make more sense as well.